### DISCUSSION TITLE
macOS GitActions Build Nightly/Tag Crash Upon Launch - Might have found the reason #5304

### DISCUSSION THREAD

**Discussion by dissidius** [rustdesk/rustdesk#5306] [General]
macOS GitActions Build Nightly/Tag Crash Upon Launch - Might have found the reason #5304

https://github.com/rustdesk/rustdesk/issues/5304

Moving it here because it was immediately closed and marked as invalid in Issues.

<img width="1783" alt="259351125-2c0619ac-f371-43e1-a02e-02896576258c" src="https://github.com/rustdesk/rustdesk/assets/10442022/aecc7467-7a3c-4cbf-842b-546a2e58cbeb">

In the screenshot in the left side is my build of rustdesk installer built locally on a mac while on the right side the one built through GitActions.

As you can see libsciter.dylib is missing on the right, the RustDesk client built through git actions, while on the left the installer i built manually/locally on a mac has it. Could this be the reason the client crashes ?

Don't know if i'm correct with opening it as an issue but i thought this was the reason and that maybe it could help.

Cheers and thanks for this awesome software!

**Comment by rustdesk** [comment]
CI is flutter version, not depend on sciter.
